crates/proof/std-fpvm/src/malloc.rs

```diff
@@ -7,11 +7,14 @@
 /// The global allocator for the program in embedded environments.
 #[cfg(any(target_arch = "mips64", target_arch = "riscv64"))]
 pub mod global_allocator {
-    use linked_list_allocator::LockedHeap;
+    use buddy_system_allocator::LockedHeap;
+
+    /// The maximum block size, as a power of two, for the buddy system allocator.
+    const HEAP_ORDER: usize = 32;
```

Comment:
Should this be configurable?

Reply:
Don't think we need it to be, but could be! Buddy system allows for anything from `2^0 .. 2^n` blocks to be allocated, and `2^32` is 4 gigabytes. There's no single allocation we ever make that's over `40MB` from what I've seen so far, so I think this # of "buckets" should be sufficient.

Reply:
Do you expect this maximum of `40MB` for a single allocation to change after Pectra with more blobs?

Reply:
@Inphi Not because of pectra, but yes. It's likely we see larger decompressed batch sizes due to block size increases on L2s, and that's the largest allocations we usually see. But each of the blobs themselves (containing _compressed_ data) are constant sized, so individually, the allocations for the blob data will remain the same.

Reply:
Also note that we're specifically talking about _individual_ allocations here. Memory usage with pectra should go up, just individual allocation size probably won't due to pectra itself.